example-gto/code/mlem/.github/workflows/gto-act-on-tags.yml

```diff
@@ -0,0 +1,32 @@
+name: Act on artifact registrations and promotions
```

Comment:
Q: why do we have two actions now? what is the difference (descriptions are the same) - is it a bug or intended? Let's update descriptions for them then please.

Reply:
Changed the name to more specific. In `mlem` branch we replace the old action with this one, so it's always the same action.